dbt/tests/data_tests/generic_tests/expect_unimputed_values_are_unchanged.sql

```diff
@@ -0,0 +1,18 @@
+{% test expect_unimputed_values_are_unchanged(
```

Comment:
I could also imagine making this a more generic "compare that two float columns are close in value" and then implement this test using that and some `row_condition` statements in the `schema.yml`. Would that be better?

Reply:
I could definitely imagine a test like that being generally useful, so I'm in favor of that (also could be fine to just wait until we need a more generic test, and then modify this one at that point).